docs/uzh.md

```diff
@@ -0,0 +1,15 @@
+# nf-core/configs: UZH Configuration
+
+All nf-core pipelines have been successfully configured for use on the UZH cluster at the insert institution here.
+
+To use, run the pipeline with `-profile UZH`. This will download and launch the [`uzh.config`](../conf/uzh.config) which has been pre-configured with a setup suitable for the UZH cluster. Using this profile, Nextflow will download a singularity image with all of the required software before execution of the pipeline.
```

Comment:
Is the profile name actually upper case? It might be better to try to keep all profile names lower case for consistency..